src/hotspot/share/cds/metaspaceShared.cpp

```diff
@@ -794,6 +795,16 @@ void MetaspaceShared::preload_and_dump_impl(StaticArchiveBuilder& builder, TRAPS
   }
 #endif
 
+  // Dummy call to load classes used at CDS runtime
+  JavaValue result(T_OBJECT);
+  Handle path_string = java_lang_String::create_from_str("", CHECK);
```

Comment:
The `path_string` will be used to create a `Path` object. I think it's better to avoid using an empty string, but use something like `"dummy.jar"` instead.

https://github.com/openjdk/jdk/blob/73ebb848fdb66861e912ea747c039ddd1f7a5f48/src/java.base/share/classes/jdk/internal/loader/ClassLoaders.java#L228-L239